### PR TITLE
Course team support in partner portal

### PR DESCRIPTION
**Pull request by mhoeber**
@catong 

@lamagnifica @srpearce  fyi

### CODE REVIEW

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -42,6 +42,16 @@ Portal, follow these steps.
 #. Select **Create New Account**. You will receive a confirmation email with
    your account access within 24 hours.
 
+===============================================
+Course Staff Support in the edX Partner Portal
+===============================================
+
+Course Teams at edX Partners can get technical support in the `edX Partner
```
Comment (catong) [inline en_us/shared/preface.rst]: lowercase "t" for "teams", lowercase "p" for "Partners"

Comment (catong) [inline en_us/shared/preface.rst]: Maybe "EdX partner course teams" instead of "Course teams at edX partners"?

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -42,6 +42,16 @@ Portal, follow these steps.
 #. Select **Create New Account**. You will receive a confirmation email with
    your account access within 24 hours.
 
+===============================================
+Course Staff Support in the edX Partner Portal
```
Comment (catong) [inline en_us/shared/preface.rst]: "Course Team Support" instead of "Course Staff Support"?

**Comment by catong** [comment]
:+1:  with a couple of minor edits

**Comment by lamagnifica** [comment]
@mhoeber please add to the preface in edx-analytics-dashboard repo also

**Pull request merged**